tests/test_relu.rs

```diff
@@ -0,0 +1,73 @@
+use ndarray::{arr1, arr3, Ix1, Ix3};
+
+use neural_zkp::relu::*;
```

Comment:
Generally the pattern in Rust is to write the tests int a `tests` module in the same file as the code being tested. See [here](https://github.com/worldcoin/semaphore-rs/blob/main/src/hash.rs#L110) for example.

Reply:
okay, will change the code structure to reflect this.